commands/@.js

```diff
@@ -8,88 +8,123 @@
   keyboard: 
   aliases: 
 CMD*/
-
-function isAdmin(){
+function isAdmin() {
   var admin_id = AdminPanel.getFieldValue({
     panel_name: "AdminInfo", // panel name
     field_name: "ADMIN_ID" // field name
   })
   return user.id == admin_id
 }
 
-function isNumeric(value){
+function canMasterRemove(value) {
+  if (value > 0) {
+    return false
+  }
+  var pvalue = -value
+  var masterRemoveableLimit = Libs.ResourcesLib.userRes("masterRemoveableLimit")
+  if (pvalue > masterRemoveableLimit.value()) {
+    return false
+  }
+
+  var masterRemoveablePoint = Libs.ResourcesLib.userRes("masterRemoveablePoint")
+  if (masterRemoveablePoint.value() - pvalue < 0) {
+    return false
+  }
+  return true
+}
+
+function removeByMaster(amount) {
```

Comment:
this funtion name is removeByMaster

But I do not see any removing in this function. Only adding:
  masterRemoveablePoint.add(amount)
  
so It is bad func name

Reply:
but basically we r removing the master removeable points as the amount will be negative only